Packs/HashiCorp-Vault/Integrations/HashiCorpVault/HashiCorpVault.py

```diff
@@ -782,6 +844,4 @@ def get_ch_secret(engine_path, secret):
     elif demisto.command() == 'hashicorp-reset-configuration':
         reset_config_command()
 except Exception as e:
-    LOG(e)
-    LOG.print_log()
```

Comment:
Since you've removed `LOG.print_log()` please replace all `LOG` calls with `demisto.debug()` - as the LOG messages will not be printed otherwise.